README.md

```diff
@@ -1 +1,5 @@
-# kanban-board
\ No newline at end of file
+Kanban board app.
+
+Tech stack:
+- Frontend: Vite, React, Typescript, SCSS, React Query
```

Comment:
Please, add more information how to run project locally

Reply:
done, will add more at later point as this project will grow